core-drawer-panel.css

```diff
@@ -145,3 +145,12 @@ polyfill-next-selector { content: ':host .narrow-layout > #main > [main]'; }
   box-sizing: border-box;
   -moz-box-sizing: border-box;
 }
+
+.noselect {
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
```

Comment:
`-khtml-user-select` is not used by any supported browsers. It's okay to drop this.